app/utils/ReceiptDataHelper.scala

```diff
@@ -85,12 +85,11 @@ class ReceiptDataHelper @Inject()(implicit val messages: MessagesApi) extends Lo
           )
         )
         case None =>
-          logger.warn("[ReceiptDataHelper][extractDeclaration] Client name missing")
+          errorLog("[ReceiptDataHelper][extractDeclaration] Client name missing")
           Left(ErrorModel(INTERNAL_SERVER_ERROR, "Client name missing"))
       }
       case Left(error) =>
-        logger.debug("[ReceiptDataHelper][extractDeclaration] Failed to retrieve customer details from vat-subscription\n" + error.message)
-        logger.warn("[ReceiptDataHelper][extractDeclaration] Failed to retrieve customer details from vat-subscription")
```

Comment:
same here

Reply:
i think this should be kept as errorLog since it is unable to retrieve the customer details altogether